File: messageGo/jsonData/service_data.go
package jsonData

import "game-message-core/proto"

type ServiceData struct {
	Id          int64             `json:"id"`
	Name        string            `json:"name"`
	AppId       string            `json:"appId"`
	ServiceType proto.ServiceType `json:"serviceType"`
	Host        string            `json:"host"`
	Port        int32             `json:"port"`
	MapId       int32             `json:"mapId"`
	Online      int32             `json:"online"`
	MaxOnline   int32             `json:"maxOnline"`
	CreatedAt   int64             `json:"createdAt"`
	UpdatedAt   int64             `json:"updatedAt"`
}

File: messageDotNet/jsonData/DaprServiceData/ServiceData.cs
using System;
using UnityEngine;

/// <summary>
/// 服务信息用于服务注册
/// </summary>
[Serializable]
internal class ServiceData
{
    public long Id;
    public string Name;
    public string APPID;
    public GameMessageCore.ServiceType ServiceType;
    public string Host;
    public int Port;
    public int MapId;
    public int Online;
    public int MaxOnline;
    public long CreatedAt;
    public long UpdatedAt;

    public string ToJson()
    {
        return JsonUtility.ToJson(this);
    }

}